src/Security/Authorization/Policy/src/AuthorizationMiddlewareContext.cs

```diff
@@ -0,0 +1,13 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.AspNetCore.Http;
+
+namespace Microsoft.AspNetCore.Authorization
+{
+    public class AuthorizationMiddlewareContext
+    {
+        public HttpContext Context { get; set; }
```

Comment:
Property name `HttpContext`?

A property called `Context` in a type called `AuthorizationMiddlewareContext` could be confusing at first.